superset/charts/api.py

```diff
@@ -683,7 +683,7 @@ def data_from_cache(self, cache_key: str) -> Response:
         f".cache_screenshot",
         log_to_statsd=False,
     )
-    def cache_screenshot(self, pk: int, **kwargs: Dict[str, bool]) -> WerkzeugResponse:
+    def cache_screenshot(self, pk: int, **kwargs: Any) -> WerkzeugResponse:
```

Comment:
The `bool` is incorrect given the logic below, 

```
rison_dict = kwargs["rison"]
window_size = rison_dict.get("window_size")
```

i.e., clearly the type of the `kwargs` value (`rison_dict`) is of type `Dict[str, ...]` rather than `bool`.

Reply:
can we maybe move rison_dict to named arg and get rid of kwargs?

Reply:
@ktmud I think it's a tad more complex that that as I believe the information is being provided by the [`@rison`](https://github.com/dpgaspar/Flask-AppBuilder/blob/3dd1eb28cb7646de2739491ca5e3e2c703ab7201/flask_appbuilder/api/__init__.py#L95) decorator.